api/types/target_health.go

```diff
@@ -25,8 +25,10 @@ import (
 type TargetHealthProtocol string
 
 const (
-	// TargetHealthProtocolTCP is a target health check protocol.
-	TargetHealthProtocolTCP TargetHealthProtocol = "TCP"
+	// TargetHealthProtocolTCP is the TCP target health check protocol.
+	TargetHealthProtocolTCP TargetHealthProtocol = "tcp"
```

Comment:
Could changing this from uppercase cause any backwards compatibility issues? Is there any cross-service communication of this protocol that could have non-equal versions?